client/src/app/+videos/+video-edit/shared/video-caption-add-modal.component.html

```diff
@@ -8,11 +8,11 @@ <h4 i18n class="modal-title">Add caption</h4>
 
     <div class="modal-body">
       <label i18n for="language">Language</label>
-      <div class="peertube-select-container">
-        <select id="language" formControlName="language" class="form-control">
-          <option></option>
-          <option *ngFor="let language of videoCaptionLanguages" [value]="language.id">{{ language.label }}</option>
-        </select>
+      <div class="peertube-ng-select-container">
```

Comment:
We still need a container with ng select?

Reply:
better safe than sorry ^^

Reply:
I removed it, now that components are wrapping it anyway.